README.md

```diff
@@ -16,14 +16,17 @@ pip install markdown-mdantic
 
 ## Usage
 
-Currently this module has only one config option: `init_code` which will
-execute **any** python code in the environment when its `__init__` is called.
+### Config options
+`init_code` will execute **any** python code in the environment when its `__init__` is called.
 In this way you can do something like adding system path, django setup etc,
 before run.
 
 **Note**: Please note that the `init_code` thing is a huge security issue if
 you run that with untrusted input. Use it with caution!
 
+`columns` is a comma-separated list of table columns to use in each table.
+The default is to use all columns `key, type, required, description, default`.
```

Comment:
While this is absolutely ok, I feel using the list type in yaml directly is more natural. Can we change the `columns` param type to be a list instead of a string? i.e. make it looks like `[key, type, require]`